data_steward/cdr_cleaner/cleaning_rules/generate_derived_tables.py

```diff
@@ -0,0 +1,361 @@
+"""
+Create Derived Tables
+
+Until now, 'observation_period', 'drug_era, 'condition_era' were populated Pre-deid stages.
+But re-creating these tables post-deid will allow OHDSI tools to run on workbench more accurately.
+"""
+# Python imports
+import logging
+
+# Third party imports
+
+# Project imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from common import CONDITION_ERA, DRUG_ERA, OBSERVATION_PERIOD, JINJA_ENV
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+POPULATE_OBS_PRD = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{dataset_id}}.{{storage_table_name}}` AS (
```

Comment:
You can remove the open parenthesis in the end here and the end of the query. Applies to all queries